src/presentational/NewClassroomFormElement.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+const NewClassroomFormElement = props => {
+  const { label, ...other } = props;
+  other.name = other.name || label.toLowerCase();
+
+  return (
+    <div className="form-group">
+      <label>{ label }</label>
```

Comment:
Styling can be fixed by putting a span around the label text. See be5ffb20726d0c0a6c874a9a11f118b77906f2eb